lib/player/player.js

```diff
@@ -736,6 +741,34 @@ shaka.player.Player.prototype.setPlaybackRate = function(rate) {
 
 
 /**
+ * @param {shaka.player.DrmSchemeInfo.Restrictions} restrictions
+ *     Restrictions object instance with custom restricions.
+ *     For example: A Restrictions instance with maxBandwidth = 700000
+ *     and minBandwidth = 200000 will restrict
+ *     playback to video tracks with bandwidth between 700000 and 200000.
+ * @export
+ */
+shaka.player.Player.prototype.setRestrictions = function(restrictions) {
+  shaka.asserts
+    .assert(restrictions instanceof shaka.player.DrmSchemeInfo.Restrictions);
```

Comment:
Asserting on the type of something that is already enforced by the compiler is not really necessary.  I suggest removing it.  Same for the runtime check below, which is not only not necessary, but if triggered, would cause the call to be silently ignored.  I think silently ignoring a call like that is likely to cause debugging headaches later.